README.md

```diff
@@ -37,7 +53,7 @@ GROQ_API_KEY=your-groq-key
 MISTRAL_API_KEY=your-mistral-key
 ```
 
-### 2. Create Workflow File
+### 3. Create Workflow File
```

Comment:
🛠️ Clear instructions for creating the workflow file! This is essential for users to set up the integration. ⚙️